nemo/collections/cv/data/video_to_text_dataset.py

```diff
@@ -0,0 +1,361 @@
+import random
+from math import isclose
+from typing import Optional
+
+import torch
+from omegaconf import DictConfig
+from omegaconf.listconfig import ListConfig
+from torch.utils.data import ChainDataset
+
+from nemo.collections.asr.data import audio_to_text
+from nemo.collections.cv.data import video_to_text
+from nemo.utils import logging
+
+
+def get_video_to_text_bpe_dataset_from_config(
+    config,
+    local_rank: int,
+    global_rank: int,
+    world_size: int,
+    tokenizer,
+    preprocessor_cfg: Optional[DictConfig] = None,
+):
+    """
+    Construct Video-To-Text BPE dataset from a config.
+    Args:
+        config: BPE dataset config
+        local_rank: model local rank
+        global_rank: model global rand
+        world_size: world size
+        tokenizer: BPE tokenizer
+        preprocessor_cfg: preprocessor config, for DALI BPE dataset
+
+    Returns:
+        constructed dataset or None if dataset config is invalid or nothing to load
+    """
+
+    is_concat = config.get('is_concat', False)
+    if is_concat:
+        if 'concat_sampling' in config and config['concat_sampling'] is None:
+            logging.warning(f"Concat dataset requires `concat_sampling` but it was not provided. Config: {config}")
+            return None
```

Comment:
I know this is a copy-paste, but I don't really understand why we return None here in any of the datasets. I think we should be raising an exception here, because currently this will just cause some downstream code to fail and it will be harder to find an issue